pubspec.yaml

```diff
@@ -10,7 +10,7 @@ authors:
   - Trent Grover <trent.grover@workiva.com>
 homepage: https://github.com/Workiva/w_flux
 dependencies:
-  react: ">=0.5.0 <2.0.0"
+  react: ">=0.5.0 <3.0.0"
```

Comment:
looks like that came in at 0.9.0, so let's go ahead and make this `react: ">=0.9.0 <3.0.0"`